flang/lib/Lower/OpenMP/ClauseProcessor.h

```diff
@@ -138,7 +138,7 @@ class ClauseProcessor {
   template <typename T>
   bool processMotionClauses(lower::StatementContext &stmtCtx,
                             mlir::omp::MapClauseOps &result);
-
+  bool processAligned(mlir::omp::AlignedClauseOps &result) const;
```

Comment:
Nit: The method list should be alphabetically sorted. In this case, right before `processAllocate`.